app/jobs/create_derivatives_job.rb

```diff
@@ -2,6 +2,15 @@
 class CreateDerivativesJob < ActiveJob::Base
   queue_as CurationConcerns.config.ingest_queue_name
 
+  after_perform do |job|
+    fileset = job.arguments.first
+    fileset_uri = fileset.uri.to_s
+    uploaded_file = Sufia::UploadedFile.find_by_file_set_uri(fileset_uri)
+    unless FileSet.where(digest_ssim: uploaded_file.checksum).empty?
+      uploaded_file.update_attribute(:status, "ingested")
```

Comment:
if we make the checksum validation only validate on :create, we can change this to `.update`